posts/post_resources.robot

```diff
@@ -0,0 +1,15 @@
+*** Setting ***
+Library    HttpLibrary.HTTP
+
+
+*** Variables ***
+${HOST}    jsonplaceholder.typicode.com
+
+${user id}     1
+${title}       This is a simple title
+${body}        This is the body of a simple post. I know, it's boring.
+
+*** Keywords ***
+
+
+
```

Comment:
I suggest creating a general resource file that is useful for all sections, not only posts. You can place there the host variable, the keywords suggested below and any other stuff that is required for all sections